client/blocks/inline-help/index.jsx

```diff
@@ -81,13 +96,68 @@ class InlineHelp extends Component {
 
 	toggleInlineHelp = () => {
 		const { showInlineHelp } = this.state;
+
 		if ( showInlineHelp ) {
 			this.closeInlineHelp();
 		} else {
 			this.showInlineHelp();
 		}
 	};
 
+	getTask = () => {
+		const task = find(
+			this.props.tasks,
+			( { id, completed } ) => ! completed && ! get( this.props.taskStatuses, [ id, 'completed' ] )
+		);
+		return task;
+	};
+
+	shouldShowChecklist = () => {
+		const { sectionName } = this.props;
+		const isAtomicSite = get( this.props, 'selectedSite.options.is_automated_transfer' );
+		const isJetpackSite = get( this.props, 'selectedSite.jetpack' );
```

Comment:
You can take advantage of `isEligibleForDotcomChecklist` selector here.
https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/is-eligible-for-dotcom-checklist.js